### PR TITLE
[query] Remove Region.{load,store}Primitive

### PR DESCRIPTION
**Pull request by chrisvittal**

### CODE REVIEW

**Comment by chrisvittal** [comment]
Let's see how CI likes it.

File: hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala [hunk 1]
```diff
@@ -203,41 +196,38 @@ class PrimitiveRVAState(val vtypes: Array[VirtualTypeWithReq], val kb: EmitClass
...
-            cb += storageType.setFieldPresent(dest, i)
-            cb += Region.storePrimitive(t, storageType.fieldOffset(dest, i))(v)
-          })
-    })
+    // everything is primitives, so this is fine! :)
+    val r = new Value[Region] {
+      def get: Code[Region] = Code._null[Region]
+    }
+    val dest = cb.newLocal("prim_rvastate_store_dest", destc)
+    storageType.storeAtAddressFromFields(cb, dest, r, fields.map(_.load), false)
```
Comment (tpoterba) [inline hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala:204]: can you just pass a `null` `Code[Region]` instead of a wrapped null?

**Review by tpoterba — CHANGES_REQUESTED** [review]
one drive by comment before full review. These changes are a breath of fresh air!

File: hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala (hunk 1 above)
Comment (chrisvittal) [reply hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala:204]: We'll try it.

**Comment by chrisvittal** [comment]
Scala tests pass. Gotta unbreak the python tests.

File: hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala (hunk 1 above)
Comment (chrisvittal) [reply hail/src/main/scala/is/hail/expr/ir/agg/AggregatorState.scala]: seems to work

**Comment by tpoterba** [comment]
nice! can you run benchmarks?

**Review by tpoterba — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**